pkgs/top-level/python-packages.nix

```diff
@@ -9264,6 +9262,25 @@ in modules // {
     };
   };
 
+  django_raster = buildPythonPackage rec {
+    name = "djongoraster-${version}";
+    version = "0.2";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/09/3d/5b9b491186579825ef6e81d60c8ddaf86d8c98d928753a329980bf48b050/django-raster-0.2.tar.gz";
```

Comment:
I was updating that when you pushed. I’ll fix it directly on master (and I just spotted a typo on the name, I used djongo instad of django…)